rules/utils/numeric.js

```diff
@@ -8,7 +8,7 @@ const isDecimalIntegerNode = node => isNumber(node) && isDecimalInteger(node.raw
 
 const isNumber = node => typeof node.value === 'number';
 const isBigInt = node => Boolean(node.bigint);
-const isNumeric = node => isNumber(node) || isBigInt(node);
+const isNumeric = node => node ? isNumber(node) || isBigInt(node) : false;
```

Comment:
Wrap the middle part in parens for readability.

Reply:
Addressed in this [commit](https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1418/commits/960ed280fcc23a1aaac0cc8c4f1e65a416241fda).